portfolio/src/main/webapp/style.css

```diff
@@ -53,21 +57,77 @@ nav ul li a{
 	color: #EEABA2;
 	font-size: 17px;
 	font-weight: bolder;
-
 }
 .active{
 	font-weight: bolder;
 	color: black;
 }
-
-
-
-h1 {
-  color: white;
-  text-align: center;
+.text-container p:nth-child(1){
```

Comment:
Should we consider having newline after each css tag definition for improved code readability (for eg. between line 64, 65)?

Reply:
here and elsewhere in this file...